docs/data-builder-intro.md

```diff
@@ -24,190 +24,32 @@ With Data Builder:
-
-* By specifying a *patch* version, your code will use the same version
-  of Data Builder.
-* By specifying a *major* or *minor* version, your code may run a newer
-  version of Data Builder, once a newer major or minor version becomes
-  available.
-  * If running locally, you can update Docker images via the
-    [OpenSAFELY CLI](opensafely-cli.md#updating-docker-images).
-
-#### Changes in the use of `latest` version
-
-!!! warning
-    Research studies often specified cohort-extractor's version as `latest`.
-
-    We no longer support specifying `latest` as a version.
-
-    This change avoids ambiguity over precisely which Data Builder version
-    was used by a given study.
-
-    If you do specify `latest` by mistake, you will instead see an error
-    mentioning `manifest unknown`. This error tells you that the version
-    is not available.
-
-## Data Builder and cohort-extractor
-
-## Missing features from cohort-extractor
-
-Data Builder is intended to eventually replace the use of
-cohort-extractor in new studies.
-
-* Many features of cohort-extractor are not yet implemented in Data
-  Builder.
-    * The current development approach is to implement a few features in Data
-      Builder fully end-to-end.
-    * See the [ehrQL reference](ehrql-reference.md) for a complete list
-      of supported features.
-* Data Builder has no current way to generate dummy data.
-    * You can supply a [CSV file containing dummy data](data-builder-dummy-data.md)
-      to Data Builder.
-    * It is possible to generate dummy data via the previous
-      [cohort-extractor](study-def-expectations.md).
-
-## The development plan for cohort-extractor
-
-cohort-extractor will continue to be supported by OpenSAFELY while Data
-Builder is in this initial design phase.
-
-Once Data Builder is ready for general use, cohort-extractor will
-continue to be maintained, where possible, so that ongoing OpenSAFELY
-studies can continue to be run.
-
-However:
+Data Builder facilitates querying multiple different data backends,
+without the researcher concerning themselves with the specific details
+of how that backend works. This means that a researcher only need 
+to write a dataset definition once and be able to use this to 
+query different datasets. 
 
-* New features are likely to only be added to Data Builder.
-* It may become infeasible to support cohort-extractor if the
-  currently supported data backends undergo considerable change.
+### Researcher-provided dummy data
+Data Builder allows researchers to provide their own dummy data to 
+use to developer their analytical code against. 
 
-### Why do study definitions and the cohort-extractor need replacing?
+## Why Data Builder was created
+For researchers familiar with OpenSAFELY, there is naturally a question as 
+to why we are writing software to replace cohort extractor. Data Builder is 
+intended to eventually replace the use of cohort-extractor in new studies.
```

Comment:
suggestion: Mention that we have a page that details more on the differences between Data Builder and cohort-extractor.